cluster.go

```diff
@@ -36,7 +37,10 @@ import (
 // consensus layer.
 var ReadyTimeout = 30 * time.Second
 
-var pingMetricName = "ping"
+const (
+	pingMetricName = "ping"
+	bootstrapCount = 3
```

Comment:
This is the number of peers we attempt to connect. We block until 3 connections succeed or we run out of addresses to try. After that the DHT will take care of discovering the swarm in the background.